Userland/Libraries/LibWeb/Layout/LayoutState.h

```diff
@@ -200,6 +200,9 @@ struct LayoutState {
         CSSPixels m_content_width { 0 };
         CSSPixels m_content_height { 0 };
 
+        CSSPixels max_width { 10000 };
+        CSSPixels max_height { 10000 };
```

Comment:
I apologise for not being clear enough that this is not the way to fix the problem. Having 10000px hard-coded constraint will break layout on pages with, for example, body size being taller than that.

In advance: increasing from 10000px to some other value won't justify this change.